gluten-core/src/main/resources/substrait/proto/substrait/algebra.proto

```diff
@@ -406,6 +406,7 @@ message RelRoot {
   Rel input = 1;
   // Field names in depth-first order
   repeated string names = 2;
+  Type.Struct output_schema = 3;
```

Comment:
Can this one upstream to substrait?

Reply:
maybe use `Type.Struct output_schema` directly and remove `repeated string names`, in `Type.Struct output_schema` it includes `names` ?

Reply:
I suggest not, don't change default upstream substrait fields to avoid possiable conflict

Reply:
It's a special case in `CH`, seems no chance to put it to upstream. https://github.com/substrait-io/substrait/issues/509

Reply:
Another way is to put the schema in the extension field. substrait dosn't limit the usage of this field.
https://github.com/oap-project/gluten/pull/1901/commits/5b63ec860affd8abd062da4fe61e55699c80c0dd